Netlens/README.rst

```diff
@@ -73,41 +73,64 @@ Installation & Usage
 ====================
 *Pre-Requisite*: Download and install CDAP_.
 
-Build the Application jar::
+From the project root, build ``Netlens`` with `Apache Maven <http://maven.apache.org/>`_ ::
+
+  $ MAVEN_OPTS="-Xmx512m" mvn clean package
+
+Note that the remaining commands assume that the ``cdap-cli.sh`` script is available on your PATH.
+If this is not the case, please add it::
+
+  $ export PATH=$PATH:<cdap-home>/bin
 
-  mvn clean package
+If you haven't already started a standalone CDAP installation, start it with the command::
 
-Deploy the Application to a CDAP instance defined by its host (defaults to localhost)::
+  $ cdap.sh start
 
-  bin/app-manager.sh --host [host] --action deploy
+On Windows, substitute ``cdap.bat`` for ``cdap.sh``.
+
+Deploy the Application to a CDAP instance defined by its host (defaults to ``localhost``)::
+  
+  $ cdap-cli.sh deploy app target/Netlens-<version>.jar
+  
+On Windows, substitute ``cdap-cli.bat`` for ``cdap-cli.sh``.
 
 Start the Application Flows and Services::
 
-  bin/app-manager.sh --host [host] --action start
+  $ cdap-cli.sh start flow Netlens.AnalyticsFlow 
+  $ cdap-cli.sh start service Netlens.AnomaliesCountService 
+  $ cdap-cli.sh start service Netlens.AnomaliesService 
+  $ cdap-cli.sh start service Netlens.CountersService 
 
 Make sure they are running::
 
-  bin/app-manager.sh --host [host] --action status
+  $ cdap-cli.sh get flow status Netlens.AnalyticsFlow 
+  $ cdap-cli.sh get service status Netlens.AnomaliesCountService 
+  $ cdap-cli.sh get service status Netlens.AnomaliesService 
+  $ cdap-cli.sh get service status Netlens.CountersService 
 
 Ingest sample traffic data::
 
-  bin/ingest-packets.sh --host [host]
+  $ bin/ingest-packets.sh [--host <hostname>]
+
+On Windows, substitute ``ingest-packets.bat`` for ``ingest-packets.sh``.
 
 Ingest sample traffic data with anomalies::
 
-  bin/ingest-anomalies.sh --host [host]
+  $ bin/ingest-anomalies.sh [--host <hostname>]
+
+On Windows, substitute ``ingest-anomalies.bat`` for ``ingest-anomalies.sh``.
 
-Run the Web UI (optionally use ``-Dcdap.host=hostname`` and ``-Dcdap.port=port`` to point to CDAP,
-localhost:10000 is used by default)::
+Run the Web UI (optionally use ``-Dcdap.host=hostname`` and ``-Dcdap.port=port`` to point to 
```

Comment:
since the rest of the tutorial assumes that you use standalone, we don't need to talk about a different cdap instance here.